en/simulation/gazebo.md

```diff
@@ -175,6 +175,75 @@ In addition to the existing cmake targets that run `sitl_run.sh` with parameters
 
 This approach significantly reduces the debug cycle time because simulator (e.g. gazebo) is always running in background and you only re-run the px4 process which is very light.
 
+## Video Streaming
+PX4 SITL for Gazebo supports UDP Video Streaming. The video from gazebo camera sensor, attached to a model, is streamed over UDP port 5600 and can be viewed remotely. The video is streamed by a gstreamer pipeline
```

Comment:
- Full stop at end. 
- "PX4 SITL for Gazebo supports UDP Video Streaming." is there a link for that. If not, then perhaps de-capitalise video and streaming - they are not proper nouns or first letter in sentence.

So what does this allow? - the ability to play video from the Gazebo simulation as though from a real camera? Can you also record and play this back in normal expected ways?

Reply:
>  the ability to play video from the Gazebo simulation as though from a real camera? 

Basically stream simulated video by UDP using GStreamer.

> Can you also record and play this back in normal expected ways?

I guess that's not the purpose but I guess you can record the video also using GStreamer.